### ISSUE TITLE
Read-only editfields aren't read by Orca screen-reader correctly

### ISSUE THREAD

**Issue by Mudb0y** [nicotine-plus/nicotine-plus#2647]
Nicotine+ version:  3.2.9
Operating System/Distribution: Arch Linux

### Describe the bug

When navigating a read-only editfield with arrow keys while using the Orca screen-reader, Orca stays silent. I know that the cursor is mooving, because when I hit an edge of the field I can hear my soundpack make the default noise, however Orca reads nothing. This bug doesn't happen often with other GTK apps so I presume this is a Nicotine specific issue.

### Expected behavior

Orca should speak the contents of the field as the cursor moves through it.

### Steps to reproduce the bug

Install the Orca screen-reader, steps depend on your chosen distribution.

On Debian-based distros like Ubuntu, Orca should be shipped by default. On Arch based systems, install via pacman then enable the speech-dispatcherd service.

Start orca, then open Nicotine+ and navigate to the log, as an example.

When focused on the editfield, press the arrow keys, for example up and down to navigate by line.

### Additional context

If needed, I can provide audio-files showing the problem.

**Comment by mathiascode** [comment]
Thanks for reporting the issue! Seems to be happening due to us disabling cursors in read-only textview, which results in Orca not knowing which line to read.

I don't see a reason to keep cursors disabled, so I'll enable them.

Let us know if there are other areas in Nicotine+ that are inaccessible to screen reader users. It would be good to know, since we rarely receive reports about this.

**Comment by slook** [comment]
@Mudb0y The fix to this issue has been implemented into the 'master' branch, which is the latest unstable version of Nicotine+ 3.3.0, see link:

https://github.com/nicotine-plus/nicotine-plus/blob/master/doc/TESTING.md

There are several methods to test Nicotine+ builds, for instance...

- ... you could [install the development build of Nicotine+ with pip](https://github.com/nicotine-plus/nicotine-plus/blob/master/doc/TESTING.md#pip) if you would like to test the latest bug fixes and new features. That would be very appreciated!

Or, alternatively...

- ... it is easy (if you are familiar with using basic `git` commands) to [run Nicotine+ from a local Git folder](https://github.com/nicotine-plus/nicotine-plus/blob/master/doc/TESTING.md#git). You might find that useful to keep up to date with the 'master' branch, as you probably want to test code commits that may look relevant to screen reader users.

Let us know if you require any support getting the development build of Nicotine+ 3.3.0 running on your system (which you should be able to install it alongside Nicotine+ 3.2.9 for doing comparisons of the user experience), if you like.

If you find any problems in terms of accessibility with Orca, and if you have the time to compose a bug report, then please do create an Issue for it to let us know about any bugs you encounter, or functions which are lacking, and we will be very happy to work on improving it.
